src/Auth/authSaga.js

```diff
@@ -0,0 +1,8 @@
+import { select, take } from 'redux-saga/effects';
+import { LOGIN_REQUEST } from './authModule';
+
+export default function* loginFlow() {
+  yield take(LOGIN_REQUEST);
+  const { authModule } = yield select();
+  localStorage.setItem('KIZUNA_USERNAME', authModule.username);
```

Comment:
I mean you are manually synchronising what's in the localStorage and what's in the store. And I don't feel this is the right way to do it.